plugins/linux-capture/xshm-input.c

```diff
@@ -308,21 +308,31 @@ static bool xshm_server_changed(obs_properties_t *props, obs_property_t *p,
 				  : xcb_setup_roots_length(xcb_get_setup(xcb));
 
 	for (int_fast32_t i = 0; i < count; ++i) {
+		char *name;
+		char name_tmp[12];
```

Comment:
why 12?

Reply:
32-bit integer generates at most 10 digits, plus one for sign. 12 bytes will hold any possible result without overflow. Also, the compiler told me to make it this big.